codegen/method.go

```diff
@@ -254,7 +254,7 @@ func NewMethod(
 	method.setValidStatusCodes()
 
 	if method.RequestType != "" {
-		hasNoBody := method.HTTPMethod == "GET" || method.HTTPMethod == "DELETE"
+		hasNoBody := method.HTTPMethod == "GET"
```

Comment:
add a test by creating an endpoint in example-gateway that is a DELETE and has a request body.

Reply:
Good point, I added a test endpoint to exercise this path